docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,13 +2,174 @@
+historically to build a case.
+
+The {siem-app} is powerful, flexible, and highly interactive. The UI elements
+are designed to be discoverable, clickable, draggable and dropable, expandable
+and collapsible, resizable, moveable, and so forth.
+
+The *{kib} Query Language (KQL) bar* is available throughout the {siem-app} for
+searching and filtering.
+
+[float]
+[[siem-overview-ui]]
+=== Overview
+
+The Overview page provides a high-level view into your security status. It can
+help surface activity that requires further investigation.
+
+Notice the *Timeline* on the right of the {siem-ui}. It is always available in its
+collapsed or expanded state. Identify items you want to dig into, and drag and
+drop them there. Those items are waiting for you when you are ready to start your
+investigation.
+
+[role="screenshot"]
+image::siem-ui.png[]
+
+[float]
+[[hosts-ui]]
+=== Hosts
+
+The Hosts view provides summary and detailed information for the hosts you are
+monitoring. You can drag and drop items of interest from the Hosts view to
+Timeline for further investigation.
+
+[role="screenshot"]
+image::hosts-ui.png[]
+
+Interactive widgets let you drill down for deeper insights:
+
+* Hosts
+* Unique IPs
+* User Authentications (success and failures)
+* Uncommon Processes
+* Events
+
+*Hosts Detail* shows you summary information for a particular host, including
+*Host ID, First Seen timestamp, Last Seen timestamp, IP and MAC addresses, OS,
+*versions, machine type, and so forth.
+
+[float]
+[[index-config-hosts]]
+==== Index configuration for hosts events
+
+The default index patterns for hosts data are `auditbeat-*``, `winlogbeat-*``,
+`filebeat-*``, and `packetbeat-*`. You can change the default index patterns in
+{kib} -> Management -> Advanced Settings -> `siem:defaultIndex`.
+
+
+[float]
+[[network-ui]]
+=== Network
+
+The Network view provides summary and detailed information for the network
+traffic you are monitoring. You can drag and drop items of interest from the
+Network view to Timeline for further investigation.
+
+[role="screenshot"]
+image::network-ui.png[]
+
+Interactive widgets let you drill down for deeper insights:
+
+* Top Talkers
+* Top DNS Domains
+* IP Details
+* Domains
+* Users
+* Transport Layer Security certs
+
+[float]
+[[network-index-config]]
+==== Index configuration for network events
+
+The default index patterns for network data are  `filebeat-*``, and
+`packetbeat-*`. You can change the default index patterns in {kib} -> Management
+-> Advanced Settings -> `siem:defaultIndex`.
+
+
+[float]
+[[timelines-ui]]
+=== Timelines
+
+Use timelines to collect evidence and research suspicious activity during an
+investigation. 
+
+You can drag and drop items that warrant further investigation onto the Timeline
+from any place in the {siem-ui}--from Overview, Hosts, Network pages, or even
+from within Timeline itself.
+
+A timeline is responsive and persists as you move through the {siem-app}
+collecting data.
+
```

Comment:
Another thing I would mention, while introducing the new timeline feature -- and is likely to blow people's mind -- is that the Timeline can display data from many sources/indices.

This is not something long time Kibana users are used to, as Discover has always been really tied to specific index patterns.